content/en/docs/guides/repository-org.md

```diff
@@ -0,0 +1,246 @@
+You can limit the impact of an issue that escaped staging testing by using Flagger's 
+[canary releases](https://docs.flagger.app/).
+
+Changes to infrastructure can be disruptive and could cause cluster-wide outages.
+These config changes can be made in steps, first you merge a change to staging,
+when the cluster is successfully reconciled, and the new cluster state passes conformance tests,
+you then promote the change to production. The promotion process is gated by PR reviews
+and end-to-end testing.
+
+## Repo per environment
+
+This approach is similar to the [monorepo](#monorepo), with some notable differences:
+
+* In the monorepo approach all team members can read the production config since Git is not designed to restrict access
+  to certain files in a repository. Having a separate repository for production means that you can grant access to
+  a subset of team members while allowing everyone to clone staging and open pull requests.
+* Promoting changes from one environment to another can be more time-consuming especially for infrastructure changes 
+  that can't be automated with Flux image updates.
+* When using the same repository for all environments, unintentional changes to production are harder to spot,
+  especially for large pull requests. Having a dedicated production repository, limits the scope of changes
+  and makes the review process less error prone.
+
+## Repo per team
+
+Assuming your organization has a dedicated platform admin team that provides Kubernetes as-a-service for other teams.
+
+The platform admin team is responsible for:
+
+* Setting up the staging and production environments.
+* Maintains the cluster addon-ons and other cluster-wide resources (CRDs, controllers, admission webhooks, etc).
+* Onboards the dev teams repositories using Flux's `GitRepository` custom resources.
+* Configures how the dev teams repositories are reconciled on each cluster using Flux's `Kustomization` custom resources.
+
+The dev teams are responsible for:
+
+* Setting up the apps definitions (Kubernetes deployments, Helm releases).
+* Configures how the apps are reconciled on each environment (Kustomize overlays, Helm values).
+* Manages the apps promotion between environments using Flux's automated image updates to Git.
+
+### Repository structure
+
+Platform admin repository example (kustomize overlays):
+
+```console
+├── teams
+│   ├── team1
+│   ├── team2
+├── infrastructure
+│   ├── base
+│   ├── production 
+│   └── staging
+└── clusters
+    ├── production
+    └── staging
+```
+
+Dev team repository example (kustomize overlays):
+
+```console
+└── apps
+    ├── base
+    ├── production 
+    └── staging
+```
+
+A complete example of this approach can be found at
+[flux2-multi-tenancy](https://github.com/fluxcd/flux2-multi-tenancy).
+
+### Delivery
+
+The delivery process is similar to the [monorepo](#delivery-management) one. The main difference is the
```

Comment:
```suggestion
The delivery process is similar to the [monorepo](#monorepo) one. The main difference is the
```

Reply:
The link points to the monorepo delivery management section